packages/examples/.storybook/manager.js

```diff
@@ -5,6 +5,8 @@ import logo from '@storybook-addon-designs/assets/logo-with-text.svg'
 import pkg from 'storybook-addon-designs/package.json'
 
 addons.setConfig({
+  showPanel: true,
```

Comment:
```suggestion
```

IIRC the default value is `true`. Simpler examples give users a more clear instruction and vision (FYI, I'll remove `showRoots` in `preview.js` because it's not used actually.)